src/arbitrary.rs

```diff
@@ -20,6 +20,7 @@ use rand::Rng;
 /// `gen` function in this crate.
 pub trait Gen : Rng {
     fn size(&self) -> usize;
+    fn reset(&mut self) {}
```

Comment:
Can you please explain this? Preferably with documentation. (Bonus points for documenting `size`.)